htdocs/core/class/html.form.class.php

```diff
@@ -7133,7 +7133,7 @@ public function showLinkToObjectBlock($object, $restrictlinksto = array(), $excl
 				'invoice'=>array('enabled'=>$conf->facture->enabled, 'perms'=>1, 'label'=>'LinkToInvoice', 'sql'=>"SELECT s.rowid as socid, s.nom as name, s.client, t.rowid, t.ref, t.ref_client, t.total as total_ht FROM ".MAIN_DB_PREFIX."societe as s, ".MAIN_DB_PREFIX."facture as t WHERE t.fk_soc = s.rowid AND t.fk_soc IN (".$listofidcompanytoscan.') AND t.entity IN ('.getEntity('invoice').')'),
 				'invoice_template'=>array('enabled'=>$conf->facture->enabled, 'perms'=>1, 'label'=>'LinkToTemplateInvoice', 'sql'=>"SELECT s.rowid as socid, s.nom as name, s.client, t.rowid, t.titre as ref, t.total as total_ht FROM ".MAIN_DB_PREFIX."societe as s, ".MAIN_DB_PREFIX."facture_rec as t WHERE t.fk_soc = s.rowid AND t.fk_soc IN (".$listofidcompanytoscan.') AND t.entity IN ('.getEntity('invoice').')'),
 				'contrat'=>array('enabled'=>$conf->contrat->enabled, 'perms'=>1, 'label'=>'LinkToContract',
-								'sql'=>"SELECT s.rowid as socid, s.nom as name, s.client, t.rowid, t.ref, t.ref_customer as ref_client, t.ref_supplier, '' as total_ht FROM ".MAIN_DB_PREFIX."societe as s, ".MAIN_DB_PREFIX."contrat as t WHERE t.fk_soc = s.rowid AND t.fk_soc IN (".$listofidcompanytoscan.') AND t.entity IN ('.getEntity('contract').')'),
```

Comment:
What mean the total amount of a contract ?
if a contract has a service from 3 month 2021-01-01 to 2021-03-01 for 10 euros 
a service for 1 month from 2021-01-01 to 2021-01-31 for 20 euros and another
from 12 month for 2021-03-01 to 2021-05-01 for 30 euros, what is the total ?

Reply:
You ask yourself complicated questions...
If you think it's easier to display only '0', users don't understand why the total of the HT lines on the contract is never displayed and think it's a bug.
A twisted question, a twisted answer. I added a tooltip...

![image](https://user-images.githubusercontent.com/2341395/109443741-9bc57400-7a3b-11eb-9d3e-f52367d48102.png)

Reply:
Users also ask why we show 200 when a contrat is 100 on 1 month and another one is 100 for 1 year.
Tooltip is a good solution.